File: OurTests/goo/test10-struct.go
package main

import "fmt"

type foo struct {
  i int
  ip *int
  f64 float64

}

func main() {
    var f foo

    f.i = 0
    f.ip = &f.i
    f.f64 = 0.0

    fmt.Printf("%d\n", f.i);
    fmt.Printf("%d\n", *f.ip);
    fmt.Printf("%f\n", f.f64);

}

File: OurTests/c/test10-struct.c
#include <stdio.h>

struct foo {
    int i;
    int* ip;
    double f64;
};

int main() {

    struct foo f;

    f.i = 0;
    f.ip = &f.i;
    f.f64 = 0.0;

    printf("%d\n", f.i);
    printf("%d\n", *f.ip);
    printf("%f\n", f.f64);

}
